src/erldns_zone_parser.erl

```diff
@@ -1050,7 +1050,7 @@ parse_json_keys_unsorted_proplists_test() ->
                     9170529505818457214552347052832728824507861128011245996056627438339703762731346681703094163316286362641501571794424157931806097889892946273849538579240359,
                     5130491166023191463112131781994138738077497356216817935415696052248528225933414267440640871636073852185344964288812312263453467652493907737029964715172561
                 ],
-                49016, 8, {{2016, 11, 14}, {11, 36, 59}}, {{2017, 2, 12}, {11, 36, 59}}}
+                49016, 8, {{2016, 11, 14}, {11, 36, 58}}, {{2017, 2, 12}, {11, 36, 58}}}
```

Comment:
changing as the behavior  in iso8601 changed https://github.com/erlsci/iso8601/compare/1.3.1...1.3.4#diff-d3e200137405c0b880f72e03b0a057327bef66474c1e28865392ea24153d9c0eL87